mlir/lib/Target/CppOutput/gridwise_convolution_implicit_gemm_v4r4.cpp

```diff
@@ -981,6 +981,7 @@ std::unique_ptr<llvm::StringRef> mlir::translateModuleToMIOpenCFlags(ModuleOp m)
   llvm::raw_string_ostream output(resultStr);
 
   for (auto f : m.getOps<FuncOp>()) {
+    output << f.getName() << "\n";
```

Comment:
@zjing14 As a reminder, now that a function name has been added to the tuning parameter output, you won't be able to just use the output as is like before. You will need to get rid of the first line in MIOpen's compilation procedure. Please feel free to ping me offline if you need further clarifications.